content/en/account_management/billing/custom_metrics.md

```diff
@@ -7,64 +7,224 @@ aliases:
+
+| Metric Name     | Tag Values                           |
+|-----------------|--------------------------------------|
+| `request.Count` | `host:A`, `endpoint:X`, `status:200` |
+| `request.Count` | `host:B`, `endpoint:X`, `status:200` |
+| `request.Count` | `host:B`, `endpoint:X`, `status:400` |
+| `request.Count` | `host:B`, `endpoint:Y`, `status:200` |
+
+
+### Effect of Adding Tags
+
+Adding tags **may not** result in more custom metrics. Your count of custom metrics usually scales with the most granular/detailed tag.
+
+Let’s suppose you’re measuring temperature in the US and you’ve tagged your temperature metric by country and region. You submit the following data to Datadog:
+
+| Metric Name   | Tag Values                         |
+|---------------|------------------------------------|
+| `temperature` | `country:USA`, `region: Northeast` |
+| `temperature` | `country:USA`, `region: Southeast` |
+
+Suppose you wanted to add the tag `City` which has three values: `NYC`, `Miami`, and `Orlando`. Adding this tag increases the number of custom metrics as it provides more detail and granularity to your dataset as shown below:
+
+| Metric Name   | Tag Values                                          |
+|---------------|-----------------------------------------------------|
+| `temperature` | `country:USA`, `region: Northeast`, `city: NYC`     |
+| `temperature` | `country:USA`, `region: Southeast`, `city: Orlando` |
+| `temperature` | `country:USA`, `region: Southeast`, `city: Miami`   |
+
+The count of custom metrics reporting from `temperature` scales with the most granular tag, `city`.
+
+Now suppose you also wanted to tag your temperature metric by `state` (which has 2 values: `NY` and `Florida`). You’re now tagging temperature by the set of tags: `country`, `region`, `state`, and `city`. Adding the state tag doesn’t increase the level of granularity already present in your dataset provided by the city tag.
+
+To obtain the temperature in Florida, you can simply recombine the custom metrics of:
+
+* `temperature:{country:USA, state:Florida, city:Orlando}`
+* `temperature{country:USA, state:Florida, city:Miami}`
+
+[1]: /developers/metrics/types/?tab=count#metric-submission-types
+[2]: /developers/metrics/types/?tab=rate#metric-submission-types
+[3]: /developers/metrics/types/?tab=gauge#metric-submission-types
+{{% /tab %}}
+{{% tab "Histogram" %}}
+
+**A HISTOGRAM metric generates five custom metrics** for each unique combination of metric name, host, and tag values to support the Agent-side aggregations: `count`, `sum`, `min`, `max`, and `avg`. [Learn more about HISTOGRAM metric type][1].
+
+Suppose you have a histogram metric measuring `request.Latency` for two endpoints `endpoint:X` and `endpoint:Y`  on a single host:
+
+{{< img src="account_management/billing/custom_metrics/request_latency_single_host.png" alt="request_latency_single_host" responsive="true" style="width:80%;">}}
```

Comment:
I'd argue that we should keep consistent tagging schemes across all metric types. Here's why: 
1. Moving from Count -> Histogram makes more sense if the tagging scheme is the same but the count of CMs is simply 5x
2. Having the same tagging scheme diagram across histogram -> distribution -> distribution with percentiles helps drive the point across that histograms/distributions are both 5x (by default) of a gauge/count/rate but the added percentile aggregations are counted differently. Having just one host and two endpoints doesn't illustrate that as clearly.

Reply:
in case of percentiles for histograms, it doesn't change how percentiles are calculated.